src/NServiceBus.Transport.Msmq.AcceptanceTests/When_deferring_a_message_while_poller_sleeps.cs

```diff
@@ -102,17 +102,24 @@ public TestTimeoutStorage(ITimeoutStorage impl, Context context)
                 return next;
             }
 
-            public async Task Store(TimeoutItem entity)
+            public async Task Store(TimeoutItem entity, TransportTransaction transaction)
             {
-                await impl.Store(entity).ConfigureAwait(false);
+                await impl.Store(entity, transaction).ConfigureAwait(false);
                 context.LongTimeoutStored = true;
             }
 
-            public Task<bool> Remove(TimeoutItem entity) => impl.Remove(entity);
+            public Task<bool> Remove(TimeoutItem entity, TransportTransaction transaction) => impl.Remove(entity, transaction);
 
             public Task<bool> BumpFailureCount(TimeoutItem timeout) => impl.BumpFailureCount(timeout);
 
-            public Task<TimeoutItem> FetchNextDueTimeout(DateTimeOffset at) => impl.FetchNextDueTimeout(at);
+            public Task<TimeoutItem> FetchNextDueTimeout(DateTimeOffset at, TransportTransaction transaction) => impl.FetchNextDueTimeout(at, transaction);
+            public TransportTransaction PrepareTransaction() => impl.PrepareTransaction();
```

Comment:
Should we call this CreateTransaction? That's what it's really doing right? Or maybe CreateAndSet